beetmoverscript/script.py

```diff
@@ -294,50 +347,93 @@ def list_bucket_objects(context, s3_resource, prefix):
 
 # move_beets {{{1
 async def move_beets(context, artifacts_to_beetmove, manifest):
```

Comment:
Something feels wrong here. I don't see why we're touching this function at all. All the "declarative artifacts" jobs are supposed to have a dedicated action scopes that looks like `project:releng:beetmover:action:push-to-nightly-declarative` which would activate the function from https://github.com/mozilla-releng/beetmoverscript/pull/182/files#diff-98ba072d44cc67fadf38dc459a83d934R101 and then `artifactMap_move_beets` function from https://github.com/mozilla-releng/beetmoverscript/pull/182/files#diff-98ba072d44cc67fadf38dc459a83d934R135. So from my perspective, nothing should've changed in `move_beets`. 

I think there may me something fishy in the rebase. Some of these changes are already in production via https://github.com/mozilla-releng/beetmoverscript/commit/e554f3a7afbe8a68085dee94ab829d2ea96e6553 I think.

Reply:
It's less of a rebase at this point and more re-applying the changes, since nothing else went in cleanly. Part of this may be me misunderstanding the existing PR, and accidentally trying to undo changes as it's compared to the current master. If so it's probably worth me throwing this away and doing it again.